### PR TITLE
Create RVRS.Load_VIP_DeathCremationPr.sql

### PR DESCRIPTION
**Pull request by sailendrasingh1**
Load_VIP_DeathCremationPr
First Draft
Date: 07/28/2023

### CODE REVIEW

File: RVRS_Staging/Database/Procedure/RVRS.Load_VIP_DeathCremationPr.sql [hunk 1]
```diff
@@ -0,0 +1,497 @@
...
+----------------------------------------------------------------------------------------------------------------------------------------------
+*/
+
+	
+PRINT '6'  + CONVERT (VARCHAR(50),GETDATE(),109)	
+					
+   
+		SELECT *
+			
+	        ,CASE WHEN ClearedComment IS NOT NULL AND MeCleared !='Y' THEN 'DimMeClearedId,ClearedComment|Warning:Value in Cleared Comment Section and ME Cleared flag are not aligned' ELSE '' END AS LoadNote_1
```
Comment (almabekwoldesenbet) [inline RVRS_Staging/Database/Procedure/RVRS.Load_VIP_DeathCremationPr.sql:224]: switch this to Warning

File: RVRS_Staging/Database/Procedure/RVRS.Load_VIP_DeathCremationPr.sql [hunk 2]
```diff
@@ -0,0 +1,497 @@
...
+	        ,CASE WHEN (MeCleared = 'N' AND MeReleased != 'N') OR (MeCleared = 'Y' AND MeReleased NOT IN ('Y', 'N')) THEN 'DimMeClearedId,DimMeReleasedId|Error:ME Released Flag and ME Cleared flag are not aligned' ELSE '' END AS LoadNote_2
```
Comment (almabekwoldesenbet) [inline RVRS_Staging/Database/Procedure/RVRS.Load_VIP_DeathCremationPr.sql:225]: Remove 
OR (MeCleared = 'Y' AND MeReleased NOT IN ('Y', 'N')

**Review by almabekwoldesenbet — CHANGES_REQUESTED** [review]
It looks good except a minor change

File: RVRS_Staging/Database/Procedure/RVRS.Load_VIP_DeathCremationPr.sql (hunk 1 above)
Comment (sailendrasingh1) [reply RVRS_Staging/Database/Procedure/RVRS.Load_VIP_DeathCremationPr.sql:224]: Fixed

File: RVRS_Staging/Database/Procedure/RVRS.Load_VIP_DeathCremationPr.sql (hunk 2 above)
Comment (sailendrasingh1) [reply RVRS_Staging/Database/Procedure/RVRS.Load_VIP_DeathCremationPr.sql:225]: Fixed

**Pull request merged**